pdal/DynamicLibrary.cpp

```diff
@@ -84,7 +95,7 @@ DynamicLibrary *DynamicLibrary::load(const std::string &name,
     void *handle = NULL;
 
 #ifdef _WIN32
-    handle = ::LoadLibraryA(name.c_str());
+    handle = ::LoadLibraryW(toNative(name).c_str());
```

Comment:
I guess I don't understand the case where this would be an issue.

Reply:
A UTF-8 encoded file path is passed in. When calling Win32 API, it needs to be converted into system string, just like in FileUtil::getDirectory

Reply:
If the file path contains Unicode encoding, it will cause problems